<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeWidget</class>
 <widget class="QWidget" name="HomeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>حاجب</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="hajebInfoLabel">
        <property name="text">
         <string> هو نظام مراقبة أبوية لمتابعة أنشطة مستخدمي الحاسب والتحكم بها وذلك من خلال الخصائص والميزات التي يوفرها. ويأتي حاجب بواجهة عربية مبسطة تسهل مهمة التعامل معه ومراقبة مستخدمي جهازك بشكل ميسر</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="margin">
         <number>3</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="hajebFeatureGroupBox">
     <property name="title">
      <string>حاجب</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="hajebfeature1Label">
          <property name="text">
           <string> - يراقب نشاط المستخدم على الانترنت ويسجل كل المواقع التي قام بزيارتها.</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="margin">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="hajebfeature2Label">
          <property name="text">
           <string> - يمنع الدخول الى المواقع التي تم تسجيلها ضمن قائمة المواقع المحظورة.</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="margin">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="hajebfeature3Label">
          <property name="text">
           <string> - يمنع استخدام البرامج والألعاب التي تم تسجيلها في قائمة البرامج المحظورة.</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="margin">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="hajebfeature4Label">
          <property name="text">
           <string> - يتحكم في وقت استخدام الانترنت ويتم جدولة الأوقات لكامل الأسبوع.</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="margin">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="hajebfeature5Label">
          <property name="text">
           <string> - يستطيع مراقبة الشاشة وأخذ لقطات منها ، وكذلك يراقب لوحة المفاتيح ويسجل كل الحروف والكلمات التي تم كتابتها على اي برنامج.</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="margin">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="hajebfeature6Label">
          <property name="text">
           <string> - يوفر سجلات متابعة لكل مستخدم على حدة.</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="margin">
           <number>3</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>14</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="icon">
       <iconset resource="CoEIAHajeb.qrc">
        <normaloff>:/images/cpanel.png</normaloff>:/images/cpanel.png</iconset>
      </attribute>
      <attribute name="title">
       <string>لوحة تحكم حاجب</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="1" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>208</width>
           <height>106</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QCheckBox" name="taskManagerCheckBox">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string> تفعيل مدير المهام</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="userButton">
           <property name="text">
            <string>إضافة مستخدم جديد</string>
           </property>
           <property name="icon">
            <iconset resource="CoEIAHajeb.qrc">
             <normaloff>:/images/adduser.png</normaloff>:/images/adduser.png</iconset>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="hajebservicesButton">
           <property name="text">
            <string>تعطيل خدمات حاجب</string>
           </property>
           <property name="icon">
            <iconset resource="CoEIAHajeb.qrc">
             <normaloff>:/images/disable.png</normaloff>:/images/disable.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="registerKeyButton">
           <property name="toolTip">
            <string>هذه النسخة مجانية</string>
           </property>
           <property name="text">
            <string>تسجيل نسخة حاجب</string>
           </property>
           <property name="icon">
            <iconset resource="CoEIAHajeb.qrc">
             <normaloff>:/images/hajibIcon.ico</normaloff>:/images/hajibIcon.ico</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="icon">
       <iconset resource="CoEIAHajeb.qrc">
        <normaloff>:/images/info.png</normaloff>:/images/info.png</iconset>
      </attribute>
      <attribute name="title">
       <string>نسخة حاجب</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="1">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QFormLayout" name="formLayout_3">
           <property name="labelAlignment">
            <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
           </property>
           <property name="formAlignment">
            <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
           </property>
           <property name="verticalSpacing">
            <number>2</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="coeiaHajebVerLabel">
             <property name="text">
              <string>version: 1.0</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="coeiaHajebLabel">
             <property name="text">
              <string>CoEIA Hajeb Control Panel</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="coeiaLSPVerLabel">
             <property name="text">
              <string>version: 1.0</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="coeiaLSPLabel">
             <property name="text">
              <string>CoEIA LSP Filter</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="coeiaKeyloggerVerLabel">
             <property name="text">
              <string>version: 1.0</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="coeiaKeyloggerLabel">
             <property name="text">
              <string>CoEIA Keylogger</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="coeiaProcessMonitorVerLabel">
             <property name="text">
              <string>version: 1.0</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="coeiaProcessMonitorLabel">
             <property name="text">
              <string>CoEIA Process Monitor</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="coeiaScreenshotMonitorVerLabel">
             <property name="text">
              <string>version: 1.0</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="coeiaScreenshotMonitorLabel">
             <property name="text">
              <string>CoEIA Screenshot Monitor</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="coeiaNetworkMonitorVerLabel">
             <property name="text">
              <string>version: 1.0</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="coeiaNetworkMonitorLabel">
             <property name="text">
              <string>CoEIA Network Monitor</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="coeiaHajebRemoverVerLabel">
             <property name="text">
              <string>version: 1.0</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLabel" name="coeiaHajebRemoverLabel">
             <property name="text">
              <string>CoEIA Hajeb Remover</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="updateButton">
         <property name="text">
          <string>تحديث حاجب</string>
         </property>
         <property name="icon">
          <iconset resource="CoEIAHajeb.qrc">
           <normaloff>:/images/refresh.png</normaloff>:/images/refresh.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="CoEIAHajeb.qrc"/>
 </resources>
 <connections/>
</ui>
